assets/src/edit-story/app/media/media3p/api/test/useMedia3pApi.js

```diff
@@ -77,27 +112,69 @@ describe('useMedia3pApi', () => {
             },
           },
           creationDate: '1234',
-          height: 100,
+          height: 3536,
           id: 'media/unsplash:1234',
           length: null,
           lengthFormatted: null,
           local: false,
-          mimeType: 'image/png',
+          mimeType: 'image/jpeg',
           poster: null,
           posterId: null,
           sizes: {
+            web_stories_thumbnail: {
+              file: 'media/unsplash:1234',
+              source_url:
+                'https://images.unsplash.com/photo-1515516089376-88db1e26e9c0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEzNzE5M30&fm=jpg&w=340&h=227&fit=max',
+              mime_type: 'image/jpeg',
+              width: 340,
```

Comment:
`web_stories_thumbnail` is defined as 150px wide, and not 340

Reply:
Commented above on our plan for image widths.